xml/abstract_commands.xml

```diff
@@ -165,7 +165,16 @@
             1: Copy data from {\tt arg0} portion of {\tt data} into the
                memory location specified in {\tt arg1}.
         </field>
-        <field name="0" bits="15:0" />
+        <field name="uncached" bits="15">
+            0: Memory accesses go through the hart's local caches just like any
+            normal hart memory access.
+
+            1: Memory accesses bypass the hart's local caches.
+
+            Implementations may decide for themselves which \Funcached values
```

Comment:
You could say this is a the RISC-V WARL field to be more precise than anthropromorphic “implementations may decide...”

Reply:
I'm happy to rephrase, but it's not WARL. It's more like write-any-check-cmderr. We haven't required anywhere that \Rcommand is read-legal.